pkg/descheduler/strategies/nodeutilization.go

```diff
@@ -149,38 +149,40 @@ func resourceUsagePercentages(nodeUsage NodeUsage) map[v1.ResourceName]float64 {
 	return resourceUsagePercentage
 }
 
-// classifyNodes classifies the nodes into low-utilization or high-utilization nodes. If a node lies between
-// low and high thresholds, it is simply ignored.
+// classifyNodes classifies the nodes into target node or probable node based on the threshold filter functions.
+// Nodes not matching both filters are simply ignored
```

Comment:
What is meaning of target node and probable node in context of node's utilization? `classifyNodes` classifies nodes based on utilization thresholds so it's still valid to use low/high keyword. How is this change related to implementation of high/low node utilization strategies?

Reply:
low and high node representation matched the `threshold` and `targetthreshold` in LowNodeUtilization. For HighNodeUtilization, as there is only one threshold - low nodes, should we consider the rest as high nodes ?
I tried to name them as target and probable to convey if a node is considered for pod eviction or its a node where the evicted pods could end

Reply:
> as there is only one threshold

for now though in the future it might be configurable as well

Given LNU still invokes `classifyNodes`, the semantic needs to stay the same. It's also up to a caller of `classifyNodes` to decide which arguments to pass. I still don't see a benefit of renaming the parameters or changing the order in which `classifyNodes` returns the values. Please, keep it as it is.